api-private.swagger.yaml

```diff
@@ -208,14 +198,45 @@ paths:
           description: The JSON was malformed.
         '422':
           description: JSON schema validation failed.
+  '/pdf-export/create-job/{uuid}':
+    post:
+      tags:
+        - pdf export
+      summary: Create a PDF export job.
+      parameters:
+        - name: uuid
+          in: path
+          type: string
+          description: The UUID fo the node to export.
+          pattern: '^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$'
+          required: true
+        - name: latest
+          in: query
+          type: boolean
+          description: Whether to use the latest revision of each page.
+          required: false
```

Comment:
needs a default to specify what value is assumed if its omitted

Reply:
Thanks; now defaults to `false`.